### PR TITLE
Move repoclosure for 3.1 from vault to stream-8

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: repoclosure/yum.conf [hunk 1]
```diff
@@ -17,34 +17,27 @@ module_platform_id=platform:el8
 [el8-baseos]
```
Comment (upadhyeammit) [inline repoclosure/yum.conf:17]: Should we also update the labels to `stream8-baseos` and likewise for other repositories? I dont know if we will also need to update it somewhere else to not break things.

Comment (ekohl) [reply repoclosure/yum.conf:17]: I wouldn't do so. If you do, start with it in nightly and then see if it works.

Comment (Odilhao) [reply repoclosure/yum.conf:17]: We probably want to wait for 3.1 to pass after we merge, after this one runs and get green, we can/should change Develop and 3.2, tagging @ekohl to see what he thinks

**Review by zjhuntin — APPROVED** [review]

File: repoclosure/yum.conf (hunk 1 above)
Comment (ekohl) [reply repoclosure/yum.conf:17]: I think I'd stick to el8. It's good enough here.

**Comment by ekohl** [comment]
Are you planning to give nightly and 3.2 the same treatment? Those have also become very unstable due to vault.

**Pull request merged**